frontend/src/actionCreators/reportingActionCreator.js

```diff
@@ -15,3 +13,21 @@ export const fetchCoreDashboard = () =>
       return dashboard_url;
     })
     .catch(() => error(Strings.ERROR));
+
+export const fetchDashboard164 = () =>
+  CustomAxios({ errorToastMessage: "Unable to fetch dashboard." })
+    .get(`${ENVIRONMENT.apiUrl + API.DASHBOARD_164}`, createRequestHeader())
+    .then((response) => {
+      const { dashboard_url } = response.data || {};
+      return dashboard_url;
+    })
+    .catch(() => error(Strings.ERROR));
+
+export const fetchDashboard165 = () =>
+  CustomAxios({ errorToastMessage: "Unable to fetch dashboard." })
+    .get(`${ENVIRONMENT.apiUrl + API.DASHBOARD_165}`, createRequestHeader())
```

Comment:
These don't dispatch any action-specific events. We could use a generic `fetchMetabaseDashboard` function instead, and provide an ID as an argument. The ID could be stored in a constant to maintain the readability.

Thoughts?

Reply:
I also had this thought, I'll update these functions.